src/main/java/de/idealo/logback/appender/RedisBatchAppender.java

```diff
@@ -86,7 +84,7 @@ private void wrapStartupRetriesOnConnectionFailures() {
                         retryExecutorService.shutdown();
                         connectionStartupCounter.incrementAndGet();
                     } catch (Exception e1) {
-                        LOG.error("retried initialization failed " + e1.getMessage());
+                        LOG.error("retried initialization failed {}", e1.getMessage());
```

Comment:
On the next change, we should change this to just logging the stack trace rather than just the message...

Reply:
I thought it had been done by purpose, so I didn't change it.